packaging/src/license/licenses.xml

```diff
@@ -31,6 +31,29 @@
         </license>
       </licenses>
     </dependency>
+    <dependency>
+      <groupId>org.apache.tomcat</groupId>
+      <artifactId>tomcat-servlet-api</artifactId>
```

Comment:
could we simply include `tomcat-jasper` that includes both `tomcat-servlet-api` and `ecj`?